### ISSUE TITLE
Work taking all my time! Argh!! (11/29 & 11/30)

### ISSUE THREAD

**Issue by anitabe404** [studydash/cards#618]
## Your Daily Accomplishment 🚀:
Still getting situated at work. My brain is pretty much fried once I'm finished. Didn't code yesterday. Probably won't do much today.

**Comment by r002** [comment]
Awesome to hear that you've started the new job! Congrats, again!! 🤗🎉🥳

Once you get more settled, would definitely love to hear more about your new adventure! Exciting times!!

**Comment by anitabe404** [comment]
@r002 Thanks. It's been going well. We should be coming up on another video chat, haha. It's been a while. I'm eager to hear about your new job too.

**Comment by r002** [comment]
@anitabe404 Definitely!  I'm game any time you're free!  Maybe this weekend, even?  Lmk!  Would love to catch up! 😀
